app/models/ext_management_system.rb

```diff
@@ -442,6 +444,9 @@ def self.schedule_destroy_queue(id, deliver_on = nil)
       :method_name => "orchestrate_destroy",
       :deliver_on  => deliver_on,
     )
+    find(id).child_managers.each do |child_manager|
+      child_manager.destroy_queue
```

Comment:
If you do need to do this here should we honor the `deliver_on` for the child managers as well?

Reply:
actually, thinking about it: The canonical entrypoint seems to be `self.destroy_queue` or the instance version. And there is no `deliver_on`. The deliver_on is only used by `orchestrate_destroy` to throttle. So, I'd actually leave it like this?!
But I can change it if you think it's better for eventualities or consistency...